pkg/aws/ec2/ec2.go

```diff
@@ -617,6 +619,55 @@ func (c *Client) GetSubnets(ctx context.Context) (ipamTypes.SubnetMap, error) {
 	return subnets, nil
 }
 
+// describeRouteTables lists all route tables
+func (c *Client) describeRouteTables(ctx context.Context) ([]ec2_types.RouteTable, error) {
```

Comment:
nit: I would have put this function above the function where it's called `GetRouteTables`.

Reply:
isn't this function just above GetRouteTables?